Doc/library/queue.rst

```diff
@@ -187,9 +187,6 @@ fully processed by daemon consumer threads.
    processed (meaning that a :meth:`task_done` call was received for every item
    that had been :meth:`put` into the queue).
 
-   ``shutdown(immediate=True)`` calls :meth:`task_done` for each remaining item
```

Comment:
It still remained in the docstring. This was also copied in `asyncio.queue`.

I wonder, was this an initial intention which was not implemented by mistake?